client/js/components/button/BackToTopButton.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <dp-button
+    :aria-label="Translator.trans('backToTop')"
+    :title="Translator.trans('backToTop')"
+    variant="outline"
+    color="primary"
+    :class="{ 'hide-visually': hide }"
+    hide-text
+    rounded
+    icon="arrow-up"
+    icon-size="large"
+    :style="buttonPosition"
+    @click="scrollTop">
+  </dp-button>
```

Comment:
```suggestion
    @click="scrollTop" />
```

Reply:
changed in 9487307521c058b398e728dc4b72f98556c11350